.github/workflows/ci.yml

```diff
@@ -7,6 +7,7 @@ concurrency:
 on:
   merge_group:
   pull_request:
+  push:
```

Comment:
```suggestion
```

Builds for `main` commit shas are covered by how the merge queue works so this is not required / results in double CI runs for each commit that lands in `main`.

Reply:
Ah, I had this so that I can verify that the tests pass. I was just about to remove it myself but you beat me to it.